ynr/templates/volunteer.html

```diff
@@ -0,0 +1,153 @@
+{% extends "base.html" %}
+{% load markdown_deux_tags %}
+{% block content %}
+  {% filter markdown%}
+<div markdown="1">
+# How to Volunteer: collect candidate information
+
+We need your help collecting candidate information. Read this guide to understand how you can help!
+
+Democracy Club Candidates is an open-source database of all candidates standing in UK elections. Its data powers
+Democracy Club’s [Who Can I Vote For?](https://whocanivotefor.co.uk/), and is regularly used by a wide range of
+organisations, including [The Electoral Commission](https://www.electoralcommission.org.uk/), social media
+companies, and national newspapers.
+
+## How to help crowdsource candidates
+
+Elections in the UK are run on a local level by individual councils. This means that the state does not collect a
+central list of candidates in a given election. This site aims to fill that gap. Hundreds of brilliant volunteers,
+like you, come together to make this data available.
+
+First you must [create your account](https://candidates.democracyclub.org.uk/accounts/signup/).
+
```

Comment:
Maybe just add something about why we require accounts (aka another thing to sign up for)?

Reply:
@pmk01 Could you provide exact text for this and I'll add?

Reply:
User accounts are necessary for transparency and security of the database - all edits are logged and recorded on each candidate profile. Edits are reviewed by Democracy Club staff, and users will be banned if vandalism is suspected.

Reply:
howzat? thoughts welcome!